crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -76,8 +108,16 @@ impl FormatNodeRule<Parameters> for FormatParameters {
                 last_node = Some(parameter_with_default.into());
             }
 
+            // Second dangling comment: trailing the slash, e.g.:
+            // ```python
+            // def f(
+            //     x,
+            //     /,  # comment
+            //     y,
+            //     z,
+            // ): ...
             let slash_comments_end = if posonlyargs.is_empty() {
-                0
+                parenthesis_comments_end
             } else {
                 let slash_comments_end = dangling.partition_point(|comment| {
```

Comment:
Should we use `dangling[parenthesis_comments_end..]` here to aovid that a comment incorrectly gets formatted twice?